server/controller/receipt_controller.py

```diff
@@ -0,0 +1,12 @@
+from datetime import date
+
+class ReceiptController:
```

Comment:
this looks like a model, rather a controller! 

other than that, this looks like it's on the right track to be a good Receipt model :)

Reply:
I think for the Models, we still need to map the receipt class to a table for the database. But the columns you created looks good! Don't forget the primary key!

Reply:
I made some changes to Receipt controller and created Receipt model that handles the actual creation of the receipt. Is this more in the right direction? Also, I haven't added the primary key yet but I'll get to it later today!